tests/test_gabbits.py

```diff
@@ -36,6 +36,8 @@ def __init__(self):
         self.environ = {
             "ROCKET_ADDRESS": XSNIPPET_API_HOST,
             "ROCKET_PORT": str(XSNIPPET_API_PORT),
+            "ROCKET_LOG": "critical",
+            "RUST_LOG": "xsnippet_api=debug",
```

Comment:
What is `xsnippet_api`? Is this the name we specify somewhere?

Reply:
Oh, it's jus the name of our crate.

Reply:
I've only enabled `debug` logging for xsnippet_api in tests, as otherwise the logs become too verbose because of the output of all the libraries.